js/docs/src/content/docs/video-metadata/c2pa-integration.md

```diff
@@ -0,0 +1,194 @@
+              "Iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/2008-02-29/"
+            },
+            "dc:creator": "did:plc:y3lae7hmqiwyq7w2v3bcb2c2",
+            "dc:title": [
+              "🦎🦎"
+            ],
+            "dc:date": [
+              "2025-10-21T19:24:24.156Z"
+            ],
+            "distributionPolicy": {
+              "deleteAfter": "2025-10-21T19:29:24.000Z"
+            },
+            "Iptc4xmpExt:LinkedEncRightsExpr": "http://creativecommons.org/publicdomain/zero/1.0/"
+          },
+          "kind": "Json"
+        },
+        {
+          "label": "place.stream.metadata.configuration",
+          "data": {
+            "$type": "place.stream.metadata.configuration",
+            "contentRights": {
+              "license": "place.stream.metadata.contentRights#cc0_1__0"
+            },
+            "distributionPolicy": {
+              "deleteAfter": 300
+            }
+          }
+        },
+        {
+          "label": "place.stream.livestream",
+          "data": {
+            "url": "https://picnic-labs-nicholas-not.trycloudflare.com",
+            "post": {
+              "cid": "bafyreicucf722xnyf74psia5ghd5usdnona4e7bkcgbqhdma2a6dokqh5m",
+              "uri": "at://did:plc:y3lae7hmqiwyq7w2v3bcb2c2/app.bsky.feed.post/3lxyfybn55m2o"
+            },
+            "$type": "place.stream.livestream",
+            "thumb": {
+              "ref": {
+                "$link": "bafkreiauoc74hcintbaua7tvp233qbfl4iymiyocc5aclhyohkz3bdinty"
+              },
+              "size": 9231,
+              "$type": "blob",
+              "mimeType": "image/jpeg"
+            },
+            "title": "🦎🦎",
+            "createdAt": "2025-10-06T16:25:06.950Z"
+          }
+        }
+      ],
+      "signature_info": {
+        "issuer": "Streamplace",
+        "common_name": "did:key:zQ3shfmFgwDstMiGaAkS4HhMJ7p3pTVhyLTHz9ABbhd4v4KJn",
+        "cert_serial_number": "54472225560857906834076190516168844896"
+      },
+      "label": "urn:c2pa:b23b55a7-bd34-4138-99d8-ce565fab3934"
+    }
+  }
+}
+```
+
+The official version of c2patool can extract this manifest, but will not
+consider it valid due to the use of ES256K. If you build c2patool from the
+[fork](https://github.com/hyphacoop/c2pa-rs/tree/es256k-rebase) used by
+Streamplace, it will validate.
+
+Note the variety of information stored in the manifest: user DID, signing key,
+timestamp, content warnings, copyright, etc. More can be added in the future,
+for example whether you consent to remixing.
+
+You can see several assertions with the name `place.stream.*`. This is 
+where Streamplace-specific metadata is stored, and is related to the lexicon.
+It's the easiest place to parse out this metadata.
+
+Some metadata (like content rights) might be duplicated in other `assertions`,
+that aren't custom and are mentioned in the C2PA spec. This allows
+C2PA-compatible software not familiar with Streamplace to still display
+information about the video content.
```

Comment:
I get what you're going for here but I think it could be phrased more clearly, something like...

```suggestion
In addition to the primary `place.stream` assertions, we make a best-effort attempt to translate the Streamplace assertions into spec-compliant C2PA metadata assertions, which are also included in the signed manifest. This allows other C2PA-compliant software to parse out information about Streamplace segments, such as content warnings. However, not everything Streamplace does fits neatly into C2PA-compliant metadata, so the primary source of truth for metadata on a Streamplace segment remains the `place.stream` assertions.
```